modules/coreos-layering-configuring.adoc

```diff
@@ -0,0 +1,186 @@
+// Module included in the following assemblies:
+//
+// * machine_management/coreos-layering.adoc
+
+:_content-type: PROCEDURE
+[id="coreos-layering-configuring_{context}"]
+= Applying a {op-system} layered image
+
+To apply a {op-system-first} layered image to your cluster, you need to create a `MachineSet` object that points to the layered image. You need a machine config for each machine config pool that you want to configure.
```

Comment:
I guess you mean here a `MachineConfig` object ?

Reply:
@EmmanuelKasper That is correct, as I understand. I made the change.